ste/mgr-JobPartTransferMgr.go

```diff
@@ -483,7 +483,15 @@ func (jptm *jobPartTransferMgr) failActiveTransfer(typ transferErrorCode, descri
 	//  consider redesign the lifecycle management in ste
 	if !jptm.WasCanceled() {
 		jptm.Cancel()
-		status, msg := ErrorEx{err}.ErrorCodeAndString()
+		serviceCode, status, msg := ErrorEx{err}.ErrorCodeAndString()
+
+		if serviceCode == "BlobUsesCustomerSpecifiedEncryption" {
+			cpkAccessFailureLogGLCM.Do(func() {
+				common.GetLifecycleMgr().Info("One or more transfers have failed because AzCopy currently does not support blobs encrypted with customer provided keys (CPK). " +
+					"If you wish to access CPK-encrypted blobs, we recommend using one of the Azure Storage SDKs to do so.")
```

Comment:
consider making into a constant in the common package, easier to find/replace afterwards